### ISSUE TITLE
Interpolation artifacts on the map boundaries in C-LINE

### ISSUE THREAD

**Issue by kdtalgo** [CEMPD/ctools#147]
The interpolation routine is causing weird features to appear on the edges of the domain in C-LINE (see attached screenshot). In some cases, like in the example provided, it skews the upper range of the contours so that the values near the roadways are under-represented in the contour coloring scheme. The boundary effect occurs for all run options (hourly, annual, cancer, etc) but skews the contour range more drastically in some cases rather than others.

![c-line_annual_cancer_risk_benzene_screenshot](https://cloud.githubusercontent.com/assets/12614252/8242707/43a815dc-15df-11e5-9ad4-6db4c4c8b68a.png)

**Comment by cseppan** [comment]
When running a single scenario (i.e. not a comparison), the values that get handed to the interpolation routine have already been converted to log scale. So these values range from -6 up to 4 or 5 (depending on the magnitude of the pollutant concentrations). The interpolator fills in zeroes around the edges since it doesn't have enough values to interpolate between. Because of the log scale, these zeroes are interpreted as 10^0 when the overlay image is created. The interpolated zeroes do not affect how the legend is drawn.

I found an old screenshot that might illustrate the problem more clearly, since the legend's range is larger than Kevin's image. The lime green "borders" around the map are the interpolated zeroes.

![tampa_new_roads](https://cloud.githubusercontent.com/assets/7329179/8318852/a5a0791e-19d6-11e5-9dc0-621dc3188e28.png)

To fix the problem, I added a step to shift the log scale concentration values into positive numbers (i.e. add 6 to every value) before interpolating the values. The edges of the image are then correctly "zero" values and don't show the visual artifacts. Here's a comparable screenshot after the fix (note that this "after" screenshot also has some other changes besides just the border fix):

![screen shot 2015-06-23 at 6 42 39 pm](https://cloud.githubusercontent.com/assets/7329179/8318957/b14665c0-19d7-11e5-9490-c166cf6afd03.png)

One other note: when reading the results from the dispersion model, the minimum allowed value is 10^-6. For annual concentrations, and especially toxics, this cuts off a lot of the data because the values are lower than this threshold. That's why Kevin's screenshot has such a small range of values.
